lib/connectors/connector.ts

```diff
@@ -16,19 +23,22 @@ export interface Connector {
   _translator: Translator;
 
   /** Client that maintains an external database connection. */
-  _client: ConnectorClient;
+  _client?: ConnectorClient | undefined;
```

Comment:
nit: `x?: Y` automatically types it with `Y | undefined` as it's optional
```suggestion
  _client?: ConnectorClient;
```